cmd/mvn.go

```diff
@@ -0,0 +1,44 @@
+/*
+
+Copyright (c) nexB Inc. and others. All rights reserved.
+ScanCode is a trademark of nexB Inc.
+SPDX-License-Identifier: Apache-2.0
+See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+See https://github.com/aboutcode-org/dependency-inspector for support or download.
+See https://aboutcode.org for more information about nexB OSS projects.
+
+*/
+
+package cmd
+
+import (
+	"github.com/aboutcode-org/dependency-inspector/internal"
+	"github.com/spf13/cobra"
+)
+
+func mvnCmd() *cobra.Command {
+	lockFiles := []string{"lockfile.json"} // Question for Code Review: Is this sufficient? Might overwrite lockfile.json in submodule
```

Comment:
we problably want to defined a more conspicuous name than just lockfile.

Reply:
I am unsure where my comments went, maybe something weird happened when I rebased it with the "-s" flag.

I have added the possibility for maven-lockfile to have a different name than "lockfile.json" in 3.5.0 and onward. This is now specified as `maven-lockfile.json` and uses maven-lockfile:5.3.5.